src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red,
```

Comment:
```suggestion
    RED,
```

ENUM contains constants, all constants are named in UPPER_CASE